### DISCUSSION TITLE
No caching in place when reopen closed file?

### DISCUSSION THREAD

**Discussion by juxeii** [clangd/clangd#1071] [Q&A] [answered]
When opening a file, the parsing starts and is eventually done.
Now, I just view the file(no editing) and close it in VSC.
Later, I reopen the file and parsing starts all over.

This means that no caching is in place for the file. This gets expensive for big files.
Would it be possible to have an option to say that the latest n files(which were once opened and parsed), where no editing took place, can be somehow cached?
That way, the "reopen" experience would be much smoother.

**Comment by HighCommander4** [comment, marked as answer]
See #312 and #942 for existing discussion of this topic
